Jamulus.pro

```diff
@@ -93,22 +93,31 @@ win32 {
 
     # replace ASIO with jack if requested
     contains(CONFIG, "jackonwindows") {
-        message(Using Jack instead of ASIO.)
-
-        !exists("C:/Program Files (x86)/Jack/includes/jack/jack.h") {
-            message(Warning: jack.h was not found at the usual place, maybe jack is not installed)
+        contains(QT_ARCH, "i386") {
```

Comment:
The indentation here is all over the place, making it very hard to read what's intended.  Please make sure only spaces are used (for example, there are leading tabs on many lines).

Reply:
Sorry, my bad.
087f835